src/main/java/com/junwoo/ott/domain/episode/entity/Episode.java

```diff
@@ -22,15 +24,22 @@
 @AllArgsConstructor(access = AccessLevel.PROTECTED)
 @SQLDelete(sql = "update episode set deleted_at = NOW() where episode_id = ?")
 @SQLRestriction(value = "deleted_at is NULL")
+@Table(uniqueConstraints = {
+    @UniqueConstraint(
+        name = "EPISODEID_TITLE_UNIQUE",
+        columnNames = {"episode_id", "title"}
```

Comment:
```suggestion
        columnNames = {"videoId", "title"}
```
제약조건 칼럼이 잘못된 것 같습니다.

Reply:
수정 했습니다.